source/renderer/app/components/layout/TopBar.js

```diff
@@ -26,13 +27,18 @@ export default class TopBar extends Component<Props> {
     ]);
 
     const topBarTitle = activeWallet ? (
-      <div className={styles.walletInfo}>
-        <div className={styles.walletName}>{activeWallet.name}</div>
-        <div className={styles.walletAmount}>
+      <span className={styles.walletInfo}>
```

Comment:
@yakovkaravelov why did we need to change `<div>`s to `<span>`s?

Reply:
@yakovkaravelov I still didn't get your answer for this question...

Reply:
@nikolaglumac 
I changed them to span to make the width of the elements shrink to content size
Needed to position badge to right edge of text content